apps/admin-gui/src/app/shared/components/set-login-dialog/set-login-dialog.component.ts

```diff
@@ -0,0 +1,70 @@
+import { Component, Inject, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { MAT_DIALOG_DATA, MatDialogRef } from '@angular/material/dialog';
+import { User, UsersManagerService } from '@perun-web-apps/perun/openapi';
+import { NotificatorService } from '@perun-web-apps/perun/services';
+import { TranslateService } from '@ngx-translate/core';
+
+export interface SetLoginDialogComponentData {
+  userId: number;
+}
+
+@Component({
+  selector: 'app-perun-web-apps-set-login-dialog',
+  templateUrl: './set-login-dialog.component.html',
+  styleUrls: ['./set-login-dialog.component.scss'],
+})
+export class SetLoginDialogComponent implements OnInit {
+  formGroup: FormGroup;
+  processing = false;
+  selectedNamespace: string;
+
+  constructor(
+    private dialogRef: MatDialogRef<SetLoginDialogComponent>,
+    @Inject(MAT_DIALOG_DATA) private data: SetLoginDialogComponentData,
+    private _formBuilder: FormBuilder,
```

Comment:
We don't use this underscore notation.

Reply:
Fixed.
(NOTE: Again I used the `create-service-member-dialog` as a template.)